yapapi/executor/__init__.py

```diff
@@ -418,7 +420,7 @@ async def worker_starter() -> None:
                     new_task = None
                     try:
                         new_task = await agreements_pool.use_agreement(
-                            lambda a: loop.create_task(start_worker(a))
+                            lambda agr, node: loop.create_task(start_worker(agr, node))
```

Comment:
if we're unfolding the abbreviation, I think I'd just go with `agreement` 
```suggestion
                            lambda agreement, node: loop.create_task(start_worker(agreement, node))
```